src/logger/fileWriter.c

```diff
@@ -280,6 +281,12 @@ static void open_log_file(struct logging_status *ls)
         pr_info(_RCP_BASE_FILE_ "Opening log file\r\n");
         ls->writing_status = WRITING_INACTIVE;
 
+        if (!sdcard_present()) {
+                ls->writing_status = SD_CARD_NOT_PRESENT;
+                pr_error(_RCP_BASE_FILE_ "SD card not present\r\n");
```

Comment:
it felt like an error condition, so...  But I can see that it could be just a normal message. But, message will show up at a higher priority, which I think is important.